rackcity/views/decommission_views.py

```diff
@@ -99,7 +160,13 @@ def decommission_asset(request):
             status=HTTPStatus.BAD_REQUEST
         )
     else:
+        
+        for assetcp in AssetCP.objects.filter(related_asset=id):
+            print(assetcp)
+            assetcp.related_decommissioned_asset = decommissioned_asset_object
```

Comment:
do you need to mark as conflict or does that happen with signals on postsave?

Reply:
i thought we were doing a separate check? the is_conflict field is only if asset is modified or deleted. you would detect this conflict by just checking if the asset has a related_decomissioned_asset -> sorry i think i forgot to flag this on your PR

Reply:
oh right I forgot 👍